airbyte-integrations/connectors/source-sendgrid/source_sendgrid/schemas/blocks.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
Why have you deleted all schemas? Isn't it easier to update them in files instead of the manifest?

Reply:
The goal is to be maintained in the connector builder and this is how that works. I was editing them in that UI.